exercises/verify.cpp

```diff
@@ -9,10 +9,25 @@ The number 3 is [not] present in the array.
 #include <iostream>
 using namespace std;
 
+bool verify(int *N,int input){
+ for (int i = 0; i < 10; i++)
+  {
+    if(N[i] == input){
+      return true;
+    }
+  }
+
+  return false;
+}
+
 int main()
 {
   // placeholder
   int N[10] = {3, 4, 5, 1, 2, 3, 4, 9, 13, 0};
```

Comment:
Invece di specificare esplicitamente il numero 10, potresti definirlo sotto gli include tramite

```c++
#define DIM 10 
```

e poi utilizzarlo sia nel ciclo for che nella definizione dell'array.

```c++
for (int i = 0; i < DIM; i++)
//....
int N[DIM] = {3, 4, 5, 1, 2, 3, 4, 9, 13, 0};
```